packages/search-engine/__tests__/search-engine.test.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
-const searchEngine = require('..');
+// const searchEngine = require('..');
```

Comment:
If you don't need this code erased, It is not a good practice, to maintain useless code in comments

Reply:
okey, I deleted this code